sass/_component_autodoc.scss

```diff
@@ -3,3 +3,36 @@ dl.class {
     margin: 0 0 0.5rem 1.5rem;
   }
 }
+
+// colour highlighting for the classes and methods
+html.no-js {
+    div.row {
+        dl.class {
```

Comment:
@Temidayo32  sorry I think there is still a gap here, please put this CSS inside the stuff under line 2.

The final full file should look like this, see how everything is nested 'inside' the `dl.class`.

I have also added some comments to a few lines.

Please update the PR and add a screenshot, also be sure to run `npm run format` to make sure your SCSS is formatted correctly.

```
dl.class {
  .sig-object + dd {
    margin: 0 0 0.5rem 1.5rem;
  }

  // colour highlighting for the classes and methods
  .sig-prename {
    color: $black;
  }

  > dt {
    font-weight: $font-weight-bold; // replace the 700 with a variable
    display: table; // try to avoid this, why not display block or something else?
    margin: 0.25rem 0;
    font-size: 90%; // let's use 0.9em instead
    line-height: normal;
    background: $gray-100;
    color: $indigo;
    border-top: 3px solid $indigo; // maybe 2px (no other borders are 3 px)
    padding: 0.25rem;
    position: relative;
  }

  &:not(.field-list) > dt {
    display: table; // if you already have display set above on the dt, you do not need it here also.
    margin-bottom: 0.25rem;// this should also have the margin bottom from the above dt
    border-left: 3px solid $gray-500; // let's use 2px instead
    background: $gray-100;
    color: $gray-600;
  }
}
```

Reply:
Ok, will do that

Reply:
Ummm.... `format` is a missing script. Tried running it.

Reply:
Sorry, `npm run lint` - I forgot this project does not use Prettier yet.
https://github.com/wagtail/sphinx_wagtail_theme/blob/0368547e4fc18d4ea6f9d095660f05b8f784bdaf/package.json#L34

Reply:
Yeah, I ran those